web-app/packages/lib/src/modules/user/views/ProfileViewTemplate.vue

```diff
@@ -5,166 +5,167 @@ SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-MerginMaps-Commercial
 -->
 
 <template>
-  <page-view
-    :style="`padding-left: ${
-      drawer ? 260 : 20
-    }px; overflow-y: auto; padding-right:20px; margin-right: 0px;`"
-  >
-    <v-layout class="column main-content fill-height">
-      <v-container>
-        <slot name="additionalBeforeContent"></slot>
-        <v-row>
-          <v-col cols="12" class="pa-0">
-            <v-card
+  <div class="profile-view">
+    <app-container
+      ><section
+        class="flex flex-column lg:flex-row lg:align-items-center row-gap-3"
+      >
+        <!-- Title with buttons -->
+        <h1 class="text-2xl text-color font-semibold">Account details</h1>
+        <div class="flex flex-grow-1 align-items-center lg:justify-content-end">
+          <PButton
+            @click="editProfileDialog"
+            data-cy="action-button-create"
+            icon="ti ti-pencil"
+            label="Edit account"
+            class="w-auto mr-1"
+            cy-data="profile-edit-btn"
+          />
+          <PButton
+            @click="changePasswordDialog"
+            severity="secondary"
+            outlined
+            cy-data="profile-change-password-btn"
+            class="w-auto"
+            icon="ti ti-lock"
+            label="Change password"
+          />
+        </div></section
+    ></app-container>
+    <app-container v-if="$slots.additionalBeforeContent"
+      ><slot name="additionalBeforeContent"></slot
+    ></app-container>
+    <app-container v-if="!loggedUser.verified_email">
+      <app-section-banner>
+        <template #header-image
+          ><img width="50" height="50" src="@/assets/warning.svg"
+        /></template>
+        <template #title>Please verify your email</template>
+        <template #description
+          >We sent you a verification email to the account you provided during
+          signup.</template
+        >
+        <template #header-actions
+          ><PButton
+            @click="resendConfirmationEmail"
+            severity="secondary"
+            outlined
+            >Send confirmation email
+          </PButton></template
+        >
+      </app-section-banner>
+    </app-container>
+    <app-container>
+      <app-section class="p-4">
+        <div
+          v-if="loggedUser"
```

Comment:
why here and not also for other actions like password change, and do we need it at all?

Reply:
cleaned